src/lib/types/minions.ts

```diff
@@ -380,6 +381,10 @@ export interface TitheFarmActivityTaskOptions extends MinigameActivityTaskOption
 export interface SepulchreActivityTaskOptions extends MinigameActivityTaskOptions {
 	type: 'Sepulchre';
 	floors: number[];
+	fletch?: {
+		fletchable: Fletchable;
```

Comment:
ActivityTaskOptions are stored in the database as json, this is storing the full fletchable object there. Like how we use monsterID instead of putting the full monster data, anything that goes in activity task options should use an ID. So this needs to be changed.

Reply:
Fixed